conga-aem-definitions/src/main/roles/aem-dispatcher.yaml

```diff
@@ -174,11 +186,6 @@ config:
     - dispatcher
     - rewrite
     - headers
-    - http2
-
-    # HTTP/2 support
```

Comment:
we should not delete this section, but keep the default to "false" and add a comment that it's enabled by adding the variant. the config section acts not only for defining the defaults, but also as documentation which configuration parameters are available.

Reply:
if i keep this section, it seems to overwrite the settings from the variant. the rendered examples with added http2-variant don't include the http2 partial then

Reply:
> we should not delete this section, but keep the default to "false" and add a comment that it's enabled by adding the variant. the config section acts not only for defining the defaults, but also as documentation which configuration parameters are available.

But we have to comment out this section. The problem is, that configurations from the main `config` block are overwriting the configurations from the variant. 
When you set a property to `true` in the variant, and to `false` in the main `config` block the result will always be `false`. I just tested this on my machine and i was not able to get a `true` value with a default set to `false`